index.html

```diff
@@ -135,6 +143,86 @@ <h2>Project name goes here</h2>
       </div>
     </section>
 
+    <section class="desktop-work">
+      <h2 class="desktop-projects">Projects</h2>
+      <div class="my-first-project">
+        <div class="project-overlay"></div>
+        <h2>Project name goes here</h2>
+        <ul>
+          <li>HTML/CSS</li>
```

Comment:
- [ ] As suggested by the previous reviewer, you need to move that project card downward and create little margin between the first project card and this one so as to match what we have on the Figma design.

### Your Design
![your-project-card](https://user-images.githubusercontent.com/26597432/236647729-b50e223a-90b4-4739-83ab-8c87df744ce5.png)

### Figma Design
![temp-project-card](https://user-images.githubusercontent.com/26597432/236647734-48bcf09c-78c8-44a8-9867-8109e2911199.png)